src/js/edu-benefits/1990/containers/EduBenefitsApp.jsx

```diff
@@ -120,7 +122,7 @@ class EduBenefitsApp extends React.Component {
         {devPanel}
         <Element name="topScrollElement"/>
         <div className="usa-width-two-thirds medium-8 columns">
-          <FormTitle title="Apply for education benefits"/>
+          {!_.includes(['introduction'], endpoint) && <FormTitle title="Apply for education benefits" subTitle="Form 22-1990"/>}
```

Comment:
It looks like this introduces a second way of telling if the page is the intro page (in addition to the later `endpoint === 'introduction'` check).